cpp/include/cudf/scalar/scalar.hpp

```diff
@@ -565,8 +566,8 @@ class chrono_scalar : public detail::fixed_width_scalar<T> {
   static_assert(is_chrono<T>(), "Unexpected non-chrono type");
 
  public:
-  chrono_scalar()  = delete;
-  ~chrono_scalar() = default;
+  chrono_scalar()           = delete;
+  ~chrono_scalar() override = default;
```

Comment:
Question: This is interesting. Why is override required here?

Reply:
It is coming from [this rule](https://clang.llvm.org/extra/clang-tidy/checks/modernize/use-override.html). In this particular case (a virtual, no-op destructor) the arguments on that page don't really apply. However, it does help in one way: it ensures a compiler error if the parent class's destructor is not declared virtual (which is UB when you have inheritance).